### ISSUE TITLE
Use native open/save dialogs [Windows]

### ISSUE THREAD

**Issue by haverholm** [fontforge/fontforge#3987]
Apparently the "open file" dialog automatically looks up the user's document folder, with the added option of browsing the disk where that folder is located. However, I keep my work files (such as font development) and private files on separate partitions, and the current GTK (?) dialog severely limits accessing the partition and files I actually need Fontforge for. I can't even generate files to my work partition.

A similar issue was raised regarding the Mac version in #3515 -- It's curious that native UI file dialogs (which actually help navigate the file system) aren't preferred over an all but useless port from Linux.

Much as I'd like to use Fontforge for font development, the hassle of just opening files makes it a very unattractive prospect.

**Comment by jtanx** [comment]
Yeah this isn't happening any time soon, it's been discussed before in other threads. It's not using GTK, the whole GUI framework is custom.

Anyway you can access files on other partitions, but to switch to a different drive it's a bit manual. Just type the drive letter into the location area like so: 
![image](https://user-images.githubusercontent.com/5137410/67469694-48e5ea80-f698-11e9-8f97-92b68f4802c4.png)

And hit enter/press ok.
